### ISSUE TITLE
Great resource--would be nice to have possibility of comments etc.

### ISSUE THREAD

**Issue by MattBrauer** [j-andrews7/Awesome-Bioinformatics-Benchmarks#7]
While this repo is a great service (and hopefully will grow into a comprehensive resource), there could be a place for comments from those who've applied the methods, or for updates and other new observations. (Maybe using the wiki functionality?)

**Comment by j-andrews7** [comment]
Yeah, I've considered that. A place where users can specify quirks/nuances/pitfalls that may be important for specific tools or methods would be useful, though I worry the organization would turn into a mess pretty quickly. If clear goals/format could be established, it'd make me a lot less hesitant to go that route.
